patches/net/minecraft/world/level/block/Blocks.java.patch

```diff
@@ -9,20 +9,13 @@
     );
     public static final Block DETECTOR_RAIL = register(
        "detector_rail", new DetectorRailBlock(BlockBehaviour.Properties.of().noCollission().strength(0.7F).sound(SoundType.METAL))
-@@ -7400,16 +_,5 @@
- 
-    public static void rebuildCache() {
-       Block.BLOCK_STATE_REGISTRY.forEach(BlockBehaviour.BlockStateBase::initCache);
--   }
--
--   static {
--      for(Block block : BuiltInRegistries.BLOCK) {
--         for(BlockState blockstate : block.getStateDefinition().getPossibleStates()) {
+@@ -7405,7 +_,8 @@
+    static {
+       for(Block block : BuiltInRegistries.BLOCK) {
+          for(BlockState blockstate : block.getStateDefinition().getPossibleStates()) {
 -            Block.BLOCK_STATE_REGISTRY.add(blockstate);
--            blockstate.initCache();
--         }
--
--         block.getLootTable();
```

Comment:
This call needs to be reintroduced due to the registry add callback not firing for vanilla entries

Reply:
I removed the removal, so actually this is adding it back. You had me worried for a bit. 😅

Reply:
Yeah, looks like I just completely misread the patch there :D